klaviyo.gemspec

```diff
@@ -2,8 +2,8 @@ files = ['klaviyo.gemspec', '{lib}/**/**/*'].map {|f| Dir[f]}.flatten
 
 Gem::Specification.new do |s|
   s.name        = 'klaviyo'
-  s.version     = '2.0.9'
-  s.date        = '2021-07-30'
+  s.version     = '2.1.0'
+  s.date        = '2021-10-01'
```

Comment:
is it possible to make a constant somewhere and then import it to the date? that way we don't need to update both the version and the user agent every release?

Reply:
I haven't found a good way to do this, the gemspec file is separate from the gem itself so I'm not sure if I can share variables between the two.  I found one way that involved another gem but even then I wasn't able to get it working.

Reply:
I did change up the variable structure a little to make it clearer though